internal/applicationsnapshot/report.go

```diff
@@ -213,6 +213,8 @@ func condensedMsg(results []conftestOutput.Result) map[string][]string {
 
 // toHACBS returns a version of the report that conforms to the
 // HACBS_TEST_OUTPUT format.
+// (Note: the name of the Tekton task result where this generally
+// gets written is now TEST_OUTPUT instead of HACBS_TEST_OUTPUT)
```

Comment:
It would be nice to update the `hacbs` references in the structs and functions here, e.g. `toTestOutput`. I'd rather the docs were written as if assuming the current state, e.g.
```
// toTestOutput returns a version of the report that conforms to the
// TEST_OUTPUT format. NOTE: This used to be called the 
// HACBS_TEST_OUPUT format.
```

Reply:
It's a good point, but I don't mind if we have vestigial traces of hacbs in the code for a while longer. Also we should think of a better name for this format.

Reply:
AppStudio Test results seems appropriate (see https://github.com/redhat-appstudio/book/blob/main/ADR/0013-integration-service-api-contracts.md)